STYLIZING.md

```diff
@@ -0,0 +1,102 @@
+icons and separators, adding newlines...
+**Please be aware that styling options should always be set before loading/sourcing P9K!
+Not every option can be changed at runtime.**
+
+## General Stylizing Options
+
+|Option|Default|Effect|
+|------|-------|------|
+|`P9K_PROMPT_ON_NEWLINE`|`false`|Display the prompt on the next line. Also checkout the sgement `newline`.|
+|`P9K_RPROMPT_ON_NEWLINE`|`false`|Display `RPROMPT` on the same line as `PROMPT` if `P9K_PROMPT_ON_NEWLINE=true`|
+|`P9K_MULTILINE_FIRST_PROMPT_PREFIX_ICON`|`?`|If the prompt is more than one line, this will be the prefix for the first line.|
+|`P9K_MULTILINE_NEWLINE_PROMPT_PREFIX_ICON`|`?`|If the prompt is at least tree lines, this will be the prefix for all lines that are neither the first nor the last.|
+|`P9K_MULTILINE_LAST_PROMPT_PREFIX_ICON`|`?`|If the prompt is more than one line, this will be the prefix for the last line.|
+|`P9K_PROMPT_ADD_NEWLINE`|`false`|Adds a newline before displaying the prompt|
+|`P9K_PROMPT_ADD_NEWLINE_COUNT`|`1`|Change the amound of newlines added if `P9K_PROMPT_ADD_NEWLINE=true`.|
+|`P9K_DISABLE_RPROMPT`|`false`|Disables `RPROMPT`.|
+|`P9K_COLOR_SCHEME`|unset|Set to `light` to invert default background and foreground color.|
+|`P9K_{LEFT,RIGHT}_SEGMENT_SEPARATOR_ICON`|`?`|This is the separator between two segments of different color.|
+|`P9K_{LEFT,RIGHT}_SUBSEGMENT_SEPARATOR_ICON`|`?`|This is the separator between two segments of the same color.|
+|`P9K_WHITESPACE_BETWEEN_{LEFT,RIGHT}_SEGMENTS`|` `|How much space there is between segment text and segment separator.|
+|``|``||
+
+## Per Segment Stylizing
+
+There are two kinds of segments stateless and stateful ones.
+This short expression describes how the variable naming scheme works:  
+Prefix `P9K_` + segment name + optionally segment state name + what you want to change:
+`P9K_<segment>[_<state>]_[BACKGROUND|FOREGROUND|BOLD|ICON[_BOLD|_COLOR]]`
+
+Detailed explanation:
+|Option|Explanation|Example|
+|------|-----------|-------|
+|`P9K_${segment}_BACKGROUND="$color"`|color for background of stateless segment|`P9K_TIME_BACKGROUND=001`|
+|`P9K_${segment}_${state}_BACKGROUND="$color"`|color for background of stateful segment|`P9K_DIR_ETC_BACKGROUND=001`|
+|`P9K_${segment}_FOREGROUND="$color"`|color for foreground of stateless segment|`P9K_TIME_BACFOREGND=2`|
+|`P9K_${segment}_${state}_FOREGROUND="$color"`|color for foreground of stateful segment|`P9K_DIR_ETC_FOREGROUND=2`|
+|`P9K_${segment}_BOLD=true`|set stateless segment to bold|`P9K_TIME_BOLD=true`|
+|`P9K_${segment}_${state}_BOLD=true`|set stateful segment to bold|`P9K_DIR_ETC_BOLD="true"`|
+|`P9K_${segment}_ICON="$icon"`|set icon/glyph for set stateless segment|`P9K_TIME_ICON="It's"`|
+|`P9K_${segment}_${state}_ICON="$icon"`|icon/glyph for set stateful segment|`P9K_DIR_ETC_ICON=$'\u2699'`|
+|`P9K_${segment}_ICON_BOLD=true`|icon/glyph/text to bold for stateless segment if font supports it|`P9K_TIME_ICON_BOLD=true`|
```

Comment:
The `/text` can be confusing as people may think it is the segment content, which is not included in this setting

Reply:
I just realized #1264 wasn't merged yet, so this probably doesn't even work right now :sweat_smile:

Reply:
Removed the `text`